src/components/BondingCurveChart/BondingCurveChart.tsx

```diff
@@ -134,14 +136,12 @@ const BondingCurveChart = observer((totalSupplyWithoutPremint:BigNumber) => {
             state: datState,
         });
 
-        
-
         if (initGoal && initGoal.gt(0)) {
             kickstarterPrice = cOrg.getPriceAtSupply(initGoal.div(2));
-            }
+        }
     
-        totalSupplyWithoutPremint = totalSupplyWithPremint.minus(initReserve);
-        currentSellPrice = reserveBalance.times(2).div(totalSupplyWithPremint);
+        totalSupplyWithoutPremint = totalSupplyWithPremint.minus(initReserve).plus(burnedSupply);
+        currentSellPrice = reserveBalance.times(2).div(totalSupplyWithPremint.plus(burnedSupply));
```

Comment:
Think this might need to look like the equation here: https://github.com/fairmint/c-org/wiki#c3-calculated-variables

Reply:
Lets test it and open another PR if we find an error with this. So far Its working good on my local tests

Reply:
According to the equation in that doc, the difference would only matter if there was burned supply.  Do you see it displaying the right price compared to the actual price from the smart contracts?  I haven't tested it yet.

Reply:
Nope, I was able to execute sells without problem and seeing the right price, I was even able to burn all initReserve tokens and sell the rest to bring the reserve to zero.

Reply:
Ok, well if it is working, I'll approve this, but am going to make an issue to revisit the sell price calculation and compare with the documentation

Reply:
See https://github.com/levelkdev/BC-DAPP/issues/297